easybuild/easyconfigs/b/breseq/breseq-0.35.2-GCC-9.3.0.eb

```
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'breseq'
+version = '0.35.2'
+
+homepage = 'https://barricklab.org/breseq'
+description = "breseq is a computational pipeline for the analysis of short-read re-sequencing data"
+
+toolchain = {'name': 'GCC', 'version': '9.3.0'}
+
+source_urls = ['https://github.com/barricklab/breseq/releases/download/v0.35.2/']
+sources = ['breseq-%(version)s-Source.tar.gz']
+checksums = ['ad9529a4a8f52e71df2e5386e8b39a66610835dcf9cbc8cec706791c2593ef4d']
+
+dependencies = [
+    ('Bowtie2', '2.4.1'),
+    ('XZ', '5.2.5'),
+    ('zlib', '1.2.11'),
+]
+
+runtest = 'test'
+
+sanity_check_paths = {
+    'files': ['bin/breseq', 'bin/gdtools'],
+    'dirs': [],
+}
+
+# breseq --help exists with non-zero exit code, so use grep
+sanity_check_commands = ["breseq --help | grep 'Usage: breseq'"]
```

It is also possible to use `breseq --version`, which returns a nice exit code 0
```suggestion
sanity_check_commands = ["breseq --version"]
```

---

That came from the existing recipe with `intel-2019a`. Should we strive for uniformity ?